pkg/prowgen/prowgen_konflux.go

```diff
@@ -127,6 +139,22 @@ func GenerateKonflux(ctx context.Context, openshiftRelease Repository, configs [
 					if err := PushBranch(ctx, r, nil, pushBranch, commitMsg); err != nil {
 						return err
 					}
+
+					for _, p := range r.PomFiles {
+						res := scrapeMaven()
+						if res.err != nil {
+							return fmt.Errorf("failed to scrape maven metadata: %w", res.err)
+						}
+						p = filepath.Join(r.RepositoryDirectory(), p)
+						if err := maven.UpdatePomFile(res.metadata, maven.RedHatMavenGA, p); err != nil {
+							return fmt.Errorf("failed to update pom file %q: %w", p, err)
+						}
+
+						commitMsg := fmt.Sprintf("[%s] Updated POM file %s with Red Hat dependencies", branchName, p)
```

Comment:
```suggestion
						commitMsg := fmt.Sprintf("[%s] Updated POM file %s with Red Hat dependencies", targetBranch, p)
```
Shouldn't really make a difference here, but theoretically on "release-next" we'd push to main